### PR TITLE
[MRG] Added copy-paste command.

### PR DESCRIPTION
**Pull request by SomeKitten**
Copy-paste is now added!

Select the gimmick to copy then right-click on the paste location and select copy-paste!

### CODE REVIEW

File: MapCanvas.py [hunk 1]
```diff
@@ -1004,6 +1007,24 @@ def button_release(self, event):
...
+
+        kwargs["name"] = gimmick.name
+
+        for i, name in enumerate(gimmick.param_names):
+            name_var = getattr(self, 'param{0}_name'.format(str(i)))
+            param_value1 = getattr(self, 'param{0}_value_1'.format(str(i)))
+            param_value2 = getattr(self, 'param{0}_value_2'.format(str(i)))
+
+            if param_value2.get():
+                kwargs[name_var.get()] = (param_value1.get(), param_value2.get())
```
Comment (monkeyman192) [inline MapCanvas.py]: doesn't this code require the code in #12 to work?

Comment (SomeKitten) [reply MapCanvas.py]: No this does not. The copy-paste standalone feature works with this code. It cannot copy *some* things, but that is a largely unrelated issue that is not related to copying and pasting itself. #12 is a minor rewrite of some of the internals of how Gimmicks are handled.

Comment (SomeKitten) [reply MapCanvas.py]: #12 does include a slightly better version of copying and pasting that I can add onto this branch. The rest of the modified code is not really copy-paste related in itself.

Comment (monkeyman192) [reply MapCanvas.py]: oh, sorry, I misread your code. All good :)

**Pull request merged**